transcoder/basisu_containers.h

```diff
@@ -26,6 +26,8 @@ namespace basisu
    {
       inline bool is_power_of_2(uint32_t x) { return x && ((x & (x - 1U)) == 0U); }
       inline bool is_power_of_2(uint64_t x) { return x && ((x & (x - 1U)) == 0U); }
+      template<class T> const T& minimum(const T& a, const T& b) { return (b < a) ? b : a; }
+      template<class T> const T& maximum(const T& a, const T& b) { return (a < b) ? b : a; }
```

Comment:
In order to avoid adding an include to basisu.h, I duplicated the functions here.

Reply:
ok I'm reviewing this